### ISSUE TITLE
Error on Ubuntu 20.04 - "ERROR! this task 'ansible.builtin.include_tasks' has extra params"

### ISSUE THREAD

**Issue by ngatti-asso** [robertdebock/ansible-role-bootstrap#47]
Hi. I just cloned the bootstrap repo and tried to run the playbook on an Ubuntu 20.04 VM. 
Browsing latest commits I saw that you tried to solve it
https://github.com/robertdebock/ansible-role-bootstrap/commit/fc282b500916cd06e70377b21a1a90e775b83df0
but it isn't working, at least on Ubuntu 20.04

Here are the details:

**Playbook:**
```
---
- name: Prepare server
  hosts: test
  gather_facts: no
  become: yes

  roles:
    - role: ansible-role-bootstrap
    - role: ansible-role-epel
```
**Command:**

`ansible-playbook  prepare.yml
`

**Error:**
```

ERROR! this task 'ansible.builtin.include_tasks' has extra params, which is only allowed in the following modules: command, raw, group_by, include_role, win_command, set_fact, meta, script, include, win_shell, import_tasks, shell, add_host, include_vars, include_tasks, import_role

The error appears to be in '/etc/ansible/roles/ansible-role-bootstrap/tasks/main.yml': line 28, column 7, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

  rescue:
    - name: gather bootstrap facts
      ^ here

```

**Environment:**

```
$cat /etc/os-release
NAME="Ubuntu"
VERSION="20.04.2 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.2 LTS"
VERSION_ID="20.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal

$ ansible --version
ansible 2.9.6
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/ansible/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  executable location = /usr/bin/ansible
  python version = 3.8.5 (default, Jan 27 2021, 15:41:15) [GCC 9.3.0]

```

**Comment by jkirk** [comment]
Just had the same error. Fixed it by updating to ansible 2.9.19 (`pip install -U "ansible<2.10"`).

**Comment by ngatti-asso** [comment]
Thanks @jkirk 
I installed ansible via the official apt repository and the latest version there is 2.9.6. I will wait until they publish a newer version.
I will close this issue as it seems an ansible problem.
